Formula/s/ssh3.rb

```diff
@@ -23,13 +22,9 @@ class Ssh3 < Formula
   uses_from_macos "libxcrypt"
 
   def install
-    system "go", "build",
-           *std_go_args(output: bin/"ssh3", ldflags: "-s -w"),
-           "cmd/ssh3/main.go"
-    ENV["CGO_ENABLED"] = "1"
```

Comment:
`CGO_ENABLED=1` was intentionally left out?

Reply:
yeah, I did

---

it was enabled due to a go bug long time ago, and it was fixed.

Reply:
something like these two:

https://redirect.github.com/hashicorp/consul/issues/16265
https://redirect.github.com/pulumi/pulumi-aws/issues/2185